docs/source/concepts.rst

```diff
@@ -440,6 +440,104 @@ matches the order of input sequence and have the exact same length.
     the adaptive micro batching feature.
```

Comment:
Remove/Update this note?

Reply:
The "API Function Return Value"  section above needs an update as well, might be good to update in a follow-up